### PR TITLE
Add files via upload + Updated Text in 30-day-trial registration flow

### PR DESCRIPTION
**Pull request by bschumac**

### CODE REVIEW

**Comment by m-mohr** [comment]
The image confuses me. It's introduced with "registration flow", but why do I need to go through NoR for example for a free tier? It also looks like something is going on with EOPlaza, but that's not in the registration flow anymore, right? I'd remove both items or place them somewhere else. EOPlaza probably where the clients are (to be used after registration) and remove NoR?

**Comment by jdries** [comment]
I would perhaps also go for a simpler visual.
I would include EGI and EOPlaza logo's perhaps next to the openEO platform logo.
The clients can be on top, in a single box.

These big arrows, NoR, Users, can all be removed to make it simpler? (Just my suggestion.)

**Comment by bschumac** [comment]
@dthiex @aljacob any suggestions from your side? Otherwise I would update this tomorrow and push it on Monday 22/05/23

**Comment by dthiex** [comment]
> @dthiex @aljacob any suggestions from your side? Otherwise I would update this tomorrow and push it on Monday 22/05/23

Will check this tomorrow morning.

File: join/free_trial.md [hunk 1]
```diff
@@ -5,15 +5,16 @@ Register for openEO platform and immediately start a 30 day free trial:
 
 <a href="https://sso.terrascope.be/auth/realms/terrascope/protocol/openid-connect/auth?client_id=openeoplatform&redirect_uri=https://openeo.cloud/welcome-to-openeo-platform/&state=0%2F95954a95-1968-4a64-8b88-fef0f47936fb&response_type=code&scope=openid" class="action-button" style="display: inline-block; font-size:1.2rem; color: #fff; background-color: #335e6f; padding: 0.8rem 1.6rem; border-radius: 4px; border-bottom: 1px solid #2e5564;">&rarr; Register for openEO Platform</a>
 
+Free trial users receive **1000 free credits** upon registration. openEO platform is a federation of services which include:
+- EGI for Authentification & Authorization via [EGI Check-in](https://www.egi.eu/services/check-in/)
+- Terrascope EOPlaza for Account management
+- EODC for provisioning of JupyterLab
+- SentinelHub, EuroDataCube and Creodiaz backends
```
Comment (dthiex) [inline join/free_trial.md:12]: Not sure what you wanted to say here but this is not correct. I would change it to: `Terrascope including Sentinel Hub connection via the EuroDataCube, EODC and Sentinel Hub backends`


- EDC is not an independent backend
- Creodia**S** is not part of the federation (you can find the updated federation graphics [here](https://github.com/openEOPlatform/documentation/blob/main/federation/federation.png))
- you forgot your backend

File: join/free_trial.md [hunk 2]
```diff
@@ -5,15 +5,16 @@ Register for openEO platform and immediately start a 30 day free trial:
 
 <a href="https://sso.terrascope.be/auth/realms/terrascope/protocol/openid-connect/auth?client_id=openeoplatform&redirect_uri=https://openeo.cloud/welcome-to-openeo-platform/&state=0%2F95954a95-1968-4a64-8b88-fef0f47936fb&response_type=code&scope=openid" class="action-button" style="display: inline-block; font-size:1.2rem; color: #fff; background-color: #335e6f; padding: 0.8rem 1.6rem; border-radius: 4px; border-bottom: 1px solid #2e5564;">&rarr; Register for openEO Platform</a>
 
+Free trial users receive **1000 free credits** upon registration. openEO platform is a federation of services which include:
```
Comment (dthiex) [inline join/free_trial.md:8]:
```suggestion
Free trial users receive **1000 free credits** upon registration. openEO Platform is a federation of services which include:
```

**Comment by dthiex** [comment]
I would also try to simplify the graphic as currently all the logos at too much confusion
- Imo for a registration flow graphic it's not important for the user to know which backends are part of openEO Platform (and we explain this already pretty nicely [here](https://docs.openeo.cloud/federation/) --> I'd remove the backends and only display that via clients/editor/jupyterlab the user gets access to the platform.

To me the graphic is not a registration flow graphic but more a federation overview and how the user interacts with this. If we want to display the registration flow I would try to go more into detail here and maybe even work with numbers to indicate different steps.

**Review by dthiex — CHANGES_REQUESTED** [review]

**Comment by bschumac** [comment]
A new proposal is available here: \openEO platform all - Dokumente\Outreach ( WP 6000 )\website

Please also feel free to update it.

**Comment by bschumac** [comment]
@jdries @m-mohr @dthiex; @aljacob; according to the discussion (also from this morning), I changed the [Registration Flow chart ](https://github.com/openEOPlatform/documentation/pull/73/commits/4ef136e54e2e8fd4507cfb8826df714545e66919).

Can you please confirm that this is as intended and I will merge this onto the docs page. 
You may also change the file, export it again and add it to this PR.

**Comment by jdries** [comment]
Fine for me! FYI, if you use draw.io, you can edit and store the original diagram files directly on github, which is very useful if we want to make changes later on. This is what we have been doing for other diagrams in this project.

File: join/free_trial.md (hunk 1 above)
Comment (bschumac) [reply join/free_trial.md]: Changed.

**Comment by dthiex** [comment]
@bschumac Thanks for updating the graphic. I find it already much clearer and certainly much better than the previous one. I still think it should be further improved to and asked our designer to propose 1-2 options. I will update the PR once I get this back. Shouldn't be more than a couple of days.

**Comment by bschumac** [comment]
@dthiex I will merge the branch now, for the updated graphic we can then create another PR.

**Pull request merged**